documentation/topics/actions/read-actions.md

```diff
@@ -58,143 +58,6 @@ query
 end
 ```
 
-## Pagination
-
-Pagination when reading records is configured on a per-action basis. Ash supports two kinds of pagination: `keyset` and `offset`.
```

Comment:
I think it might be worth leaving the section here but just redirecting to to the other guide